File: outros/ws - pycharm/primeiro/crescente.py
print("Digite dois números: ")
x = int(input())
y = int(input())

while x != y:
    if x > y:
        print("Decrescente!")
        print("Digite outros dois números: ")
        x = int(input())
        y = int(input())
    else:
        print("Crescente!")
        print("Digite outros dois números: ")
        x = int(input())
        y = int(input())

File: outros/eclipse/exercicios/src/crescente.java
import java.util.Locale;
import java.util.Scanner;

public class crescente {
	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		System.out.println("Digite dois números: ");
		int x = sc.nextInt();
		int y = sc.nextInt();
		
		do {
			if (x > y) {
				int troca = x;
				x = y;
				y = troca;
				System.out.println("Decrescente");
				System.out.println("Digite outros dois números");
				x = sc.nextInt();
				y = sc.nextInt();
			}
			else {
				System.out.println("Crescente");
				System.out.println("Digite outros dois números");
				x = sc.nextInt();
				y = sc.nextInt();
			}
		} while (x != y);
		
		sc.close();
		
	}
}
		